tests/presentations/indexable-post-type-presentation/og-article-modified-time-test.php

```diff
@@ -49,6 +49,12 @@ public function test_generate_og_article_modified_time_differs_from_published_ti
 			'post_modified_gmt' => '2019-11-09T12:34:56+00:00',
 		];
 
+		$this->date_helper
+			->expects( 'mysql_date_to_w3c_format' )
+			->with( '2019-11-09T12:34:56+00:00' )
+			->once()
```

Comment:
The order of these arguments is opposite from above (there you use `once`, then `with`).